package/abs/rancher-abs

```diff
@@ -0,0 +1,464 @@
+#!/bin/bash
+if [ -e "$(dirname $0)/common.sh" ]; then
+    source $(dirname $0)/common.sh
+elif [ -e "$(dirname $0)/../common/common.sh" ]; then
+    source $(dirname $0)/../common/common.sh
+fi
+
+readonly ACCESS_KEY_ID=${ECS_ACCESS_KEY_ID}
```

Comment:
What's this `ECS_ACCESS_KEY_ID` and `SECRET`? I didn't see it mentioned anywhere.